### PR TITLE
OSDOCS-7183: updates 4.14 docs that are impacted by OVN-K i/c archite…

### PR DESCRIPTION
**Pull request by JoeAldinger**
Updates 4.14 docs that are impacted by OVN-K i/c architecture change

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-7183

Link to docs preview:
[IPsec](https://65876--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn#nw-ovn-ipsec-verification_configuring-ipsec-ovn)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-ovn-ipsec-verification.adoc [hunk 1]
```diff
@@ -14,26 +14,22 @@ As a cluster administrator, you can verify that IPsec is enabled.
...
+ovnkube-control-plane-66564dbd96-wvkgf   2/2     Running   2 (25m ago)   34m
 ----
 
 . Verify that IPsec is enabled on your cluster:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> ovn-nbctl --no-leader-only get nb_global . ipsec
```
Comment (tssurya) [inline modules/nw-ovn-ipsec-verification.adoc:32]: please note that there is no ovnkube-master on 4.14 @JoeAldinger 
its ovnkube-node pod and that also tells me we need to be showing the ovnkube-node pods in the output?

**Review by tssurya — CHANGES_REQUESTED** [review]

File: modules/nw-ovn-ipsec-verification.adoc [hunk 2]
```diff
@@ -10,30 +10,29 @@ As a cluster administrator, you can verify that IPsec is enabled.
 
 .Verification
 
-. To find the names of the OVN-Kubernetes control plane pods, enter the following command:
+. To find the names of the OVN-Kubernetes data plane pods, enter the following command:
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master
+$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-node
```
Comment (JoeAldinger) [inline modules/nw-ovn-ipsec-verification.adoc:17]: Discussed and updated based on conversation with @anuragthehatter via slack. `nbdb` is now on data plane pods not on control plane so the procedural step needed to change.

Comment (anuragthehatter) [reply modules/nw-ovn-ipsec-verification.adoc:17]: Thanks @JoeAldinger So grep seems like a old school way. May be we can do following. Bascially pods with label `app=ovnkube-node`
```
oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-node
```

Comment (anuragthehatter) [reply modules/nw-ovn-ipsec-verification.adoc:17]: rest lgtm. thanks

**Review by sjhala-ccs — COMMENTED** [review]
Your changes LGTM but I am wondering why the verification steps are in a separate module and not part of the "Enabling IPsec encryption" procedure. 
I think the two modules should be merged but leaving it up to you to decide if that's something you want to address in this PR.

**Review by tssurya — APPROVED** [review]
/lgtm

File: modules/nw-ovn-ipsec-verification.adoc [hunk 3]
```diff
@@ -10,30 +10,29 @@ As a cluster administrator, you can verify that IPsec is enabled.
...
+ovnkube-node-zfvcl                       8/8     Running   0              34m
 ----
 
 . Verify that IPsec is enabled on your cluster:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-node-<XXXXX> ovn-nbctl --no-leader-only get nb_global . ipsec
```
Comment (JoeAldinger) [inline modules/nw-ovn-ipsec-verification.adoc:35]: @sjhala-ccs I'm not sure I follow. The verification step is part of `modules/nw-ovn-ipsec-verification`

Comment (JoeAldinger) [inline modules/nw-ovn-ipsec-verification.adoc:35]: @sjhala-ccs I'm not sure I follow. The verification step is part of `modules/nw-ovn-ipsec-verification`

**Comment by sjhala-ccs** [comment]
@JoeAldinger My question is why is `Verfiying that IPsec is enabled` a separate module with no `.Procedure` section? According to the [Modular documentation reference guide](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs), the `.Verification` section can be added after the procedure steps in the same module, but I don't think `.Verification` can be a standalone module. 

So, my suggestion was to move the verification steps to the `modules/nw-ovn-ipsec-enable.adoc` file and delete the `modules/nw-ovn-ipsec-verification.adoc` file.

I hope that helps clarify things.

**Comment by JoeAldinger** [comment]
> @JoeAldinger My question is why is `Verfiying that IPsec is enabled` a separate module with no `.Procedure` section? According to the [Modular documentation reference guide](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs), the `.Verification` section can be added after the procedure steps in the same module, but I don't think `.Verification` can be a standalone module.
> 
> So, my suggestion was to move the verification steps to the `modules/nw-ovn-ipsec-enable.adoc` file and delete the `modules/nw-ovn-ipsec-verification.adoc` file.
> 
> I hope that helps clarify things.
I agree and will take care of this after GA. Thank you.
https://issues.redhat.com/browse/OCPBUGS-21726

**Pull request merged**